static-container-registry.py

```diff
@@ -79,13 +79,15 @@ def find_images(root):
                 LOGGER.info('Invalid schemaVersion in %s', manifest)
                 continue
 
-            if data.get('mediaType') != \
-                    'application/vnd.docker.distribution.manifest.v2+json':
+            mediaType = data.get('mediaType')
+            if not mediaType in [
+                    'application/vnd.docker.distribution.manifest.v2+json',
+                    'application/vnd.oci.image.manifest.v1+json']:
                 LOGGER.info('Invalid mediaType in %s', manifest)
                 continue
 
             LOGGER.info('Found image %s:%s in %s', name, tag, curr)
-            yield (name, tag)
+            yield (name, tag, data.get('mediaType'))
```

Comment:
```suggestion
            yield (name, tag, mediaType)
```
I think?

Reply:
yes... i added the variable on line 82 for this purpose, and forgot to change it 🤦‍♂️